misc/completion/qutebrowser.zsh

```diff
@@ -0,0 +1,34 @@
+#compdef qutebrowser
+
+local ret=1
+
+_arguments -s : \
+  '(- *)'{-h,--help}'[show the help message]' \
+  '(- *)'{-V,--version}'[show version information]' \
+  '(-h --help -V --version --basedir)--basedir[base directory for all storage.]:storage base directory:_directories' \
+  '(-h --help -V --version)'-{s,-set}'[set a temporary setting for this session]:option:' \
+  '(-h --help -V --version -r --restore)'-{r,-restore}'[restore a named session]:session name:' \
+  '(-h --help -V --version -R --override-restore)'-{R,-override-restore}'[do not restore a session]' \
+  '(-h --help -V --version --target)--target[how to open URLs in running instance]:open URLs in:(auto tab tab-bg tab-silent tab-bg-silent window)' \
+  '(-h --help -V --version --backend)--backend[which backend to use]:backend:(webkit webengine)' \
+  '(-h --help -V --version --enable-webengine-inspector)--enable-webengine-inspector[enable web inspector for QtWebEngine]' \
```

Comment:
Please mention that this is a security risk, like in the manpage (I'm okay with not linking to the bug, but I think it deserves being mentioned).